### PR TITLE
Update insider-risk-management-solution-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated licensing information for Insider Risk Management

Per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9538

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/insider-risk-management-solution-overview.md [hunk 1]
```diff
@@ -35,6 +35,22 @@ Watch the videos below to learn how insider risk management can help your organi
...
+## Subscriptions and licensing
+
+Before you get started with insider risk management, you should confirm your [Microsoft 365 subscription](https://www.microsoft.com/microsoft-365/compare-all-microsoft-365-plans) and any add-ons. To access and use insider risk management, your organization must have one of the following subscriptions or add-ons:
+
+- Microsoft 365 E5/A5/F5/G5 subscription (paid or trial version)
+- Microsoft 365 E3/A3/F3/G3 subscription + the Microsoft 365 E5/A5/F5/G5 Compliance add-on
+- Microsoft 365 E3/A3/F3/G3 subscription + the Microsoft 365 E5/A5/F5/G5 Insider Risk Management add-on
+- Office 365 E3 subscription + Enterprise Mobility and Security E3 + the Microsoft 365 E5 Compliance add-on
+
+Users included in insider risk management policies must be assigned one of the licenses above.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/insider-risk-management-solution-overview.md]:
```suggestion
Users included in insider risk management policies must be assigned one of these licenses.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@msbemba Gentle reminder

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
Closing as per the comment in #9538

**Pull request closed without merging**